Language/Variables/Data Types/String/Functions/charAt.adoc

```diff
@@ -34,7 +34,7 @@ string.charAt(n)
 === Parameters
 `string`: a variable of type String
 
-`n`: a variable of type String
+`n`: a variable of type int

```

Comment:
It's `unsigned int`, not `int`:
https://github.com/arduino/ArduinoCore-avr/blob/58006613a7c858417881744419c5da54cc21a401/cores/arduino/WString.h#L156
```cpp
char charAt(unsigned int index) const;
```

Reply:
Actually you are right, I just merged by mistake, going to fix it.